lib/dpkg/trigdeferred.c

```diff
@@ -136,6 +138,7 @@ trigdef_update_start(enum trigdef_update_flags uf)
 
 		setcloexec(fileno(trig_new_deferred), newfn.buf);
 	}
+	free(triggersdir);
 
```

Comment:
These were fixed with commit 35d024ea78135d48ee778024dc61486db6db594d, as the variable cannot be freed as it needs to be used by the _done function.